participants/minh_lengoc.json

```diff
@@ -0,0 +1,42 @@
+// Please provide your info in your own .json file.
+// See https://jscraftcamp.org/registration for more information
+{
+	// your name / nickname (required)
+	"name": "Minh Le Ngoc",
+	// optional company name (required)
+	"company": "Ambright",
+	// either both days or at least one day has to be set to true
+	"when": {
+		// 2023-06-30
+		"friday": true,
+		// 2023-07-01
+		"saturday": false
+	},
+	// if you are willing to take session notes and publish them to github (required)
+	"iCanTakeNotesDuringSessions": false,
+	// your current interests (JS and in general) (required)
+	"tags": ["ReScript", "Svelte", "Zustand", "GraphQL"],
```

Comment:
Could you put some more individual tags for you in here? Looks like the defaults we had here